tests/mocha/event_test.js

```diff
@@ -997,10 +997,6 @@ suite('Events', function() {
             this.eventsFireSpy, 0, Blockly.Events.BlockDelete,
             {oldXml: expectedOldXml, group: ''},
             workspaceSvg.id, expectedId);
-        assertNthCallEventArgEquals(
```

Comment:
Is this removal intentional?

Reply:
Yes, they are the exact same call. Probably a rebase error.